hack/local-up-cluster.sh

```diff
@@ -680,8 +677,13 @@ EOF
 fi
 }
 
+# validate that etcd is: not running, in path, and has minimum required version.
+kube::etcd::validate
+
 if [[ "${CONTAINER_RUNTIME}" == "docker" ]]; then
-  test_docker
+  # source to get common docker test
+  #source "${KUBE_ROOT}/build-tools/common.sh"
+  ${KUBE_ROOT}/build-tools/common.sh kube::build::ensure_docker_daemon_connectivity
```

Comment:
a few notes:
1. this is now `build/` instead of `build-tools/`.
2. does this actually work as-written? (rather than sourcing common.sh)
3. I'd also be fine with having this function moved into `hack/lib/common.sh`. I think both `build/common.sh` and this script (`hack/local-up-cluster.sh`) include `hack/lib/`.